### PR TITLE
✨ Add Calibre and WPT to Readme

### PR DESCRIPTION
**Pull request by benschwarz**
Adds a "Lighthouse Integrations" section to `README.md` that mentions Calibre and WPT.

@pmeenan, I pinched this text from the WPT about page, but thought you might have some edits you'd like to see in this PR?

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -229,6 +229,12 @@ yarn closure
 yarn compile-devtools
 ```
 
+## Lighthouse Integrations
+
+* **[Calibre](https://calibreapp.com)** Calibre is a web performance monitoring tool running Lighthouse continuously or on-demand via an API. Test using emulated devices and connection speeds from a number of geographical locations. Set budgets and improve performance with actionable guidelines.
+
+* **[Web Page Test](https://www.webpagetest.org)** — WebPagetest is an open source project that is primarily being developed and supported by Google as part of our efforts to make the web faster.
```
Comment (ebidel) [inline readme.md]: How about:

```
- an [open source](https://github.com/WPO-Foundation/webpagetest) tool for measuring and analyzing the performance of web pages on real devices. Users can choose to produce a Lighthouse report alongside the analysis of WebPageTest results.
```

Comment (pmeenan) [inline readme.md]: LGTM

**Comment by benschwarz** [comment]
> what's the difference between integrations and related projects?

>Related projects are extensions of Lighthouse vs projects/products that use lighthouse as one of their features? Or...something? :)

The idea is that these things run Lighthouse remotely, whereas the extensions / related projects still require setup by developers.

File: readme.md [hunk 2]
```diff
@@ -229,6 +229,12 @@ yarn closure
 yarn compile-devtools
 ```
 
+## Lighthouse Integrations
+
+* **[Calibre](https://calibreapp.com)** Calibre is a web performance monitoring tool running Lighthouse continuously or on-demand via an API. Test using emulated devices and connection speeds from a number of geographical locations. Set budgets and improve performance with actionable guidelines.
```
Comment (brendankenny) [inline readme.md]: need to settle on a hyphenation scheme, and maybe bold the related project names too?

Comment (benschwarz) [inline readme.md]: @brendankenny PTAL

**Comment by brendankenny** [comment]
oh, sorry, I meant since the Calibre and WPT were getting bolded, maybe `webpack-lighthouse-plugin` et al should be bolded too

**Comment by benschwarz** [comment]
Erm. That's what I get for editing pull requests on an iPhone 😂
Got it this time, I think

**Pull request merged**